### ISSUE TITLE
net.sf.json is quite outdated

### ISSUE THREAD

**Issue by jschmied** [amzn/amazon-pay-api-sdk-java#5]
This library is not updated since 2010.
Please consider moving to a maintained library:

https://www.json.org/json-en.html
https://github.com/FasterXML/jackson

**Comment by rougou** [comment]
Yes this library is very obscure and hard to customize. We ended up converting our POJO to a string first with jackson, then converting to JSONObject.
Providing POJOs to begin with like the v1 SDK would be the best option, I think...
